test/cypress/integration/sidebar-extension.spec.ts

```diff
@@ -71,5 +77,27 @@ context('Sidebar extension', () => {
     })
   })
 
+  it('creates, gets, and deletes a content type through spaceAPI', () => {
+    cy.getSdk(iframeSelector).then(async (sdk: SidebarExtensionSDK) => {
```

Comment:
I personally don't like the mix of `.then` and `async`. Either one or the other.

Reply:
It's not possible to `await` the `getSdk` call, sorry!